scripts/zones/Wajaom_Woodlands/DefaultActions.lua

```diff
@@ -0,0 +1,5 @@
+local ID = require("scripts/zones/Wajaom_Woodlands/IDs")
+
+return {
+    ['Giwahb_Watchtower']         = { special = ID.text.GIWAHB_WATCHTOWER_LOCKED },
```

Comment:
Can remove the spaces before `=` here, since there is no other NPC requiring alignment